### DISCUSSION TITLE
Importing data from CSV and switching to another table breaks the UI

### DISCUSSION THREAD

**Discussion by zigavidmar** [supabase/supabase#16941] [Contribute to Supabase]
## Describe the bug
As I am importing data from CSV to any existing table, data gets imported but when i switch the table view, application crashes. I then need to refresh the page each time.

## To Reproduce
1. Go to random table.
2. Import data from CSV
3. Switch tables

## System information

OS: Windows
Browser: Chrome

## Additional context
There is no network errors so it's probably a typescript code issue. I think if you wait a couple of seconds it actually won't break the UI. So it could be related to some timeout.

**Comment by ajinkyalanjewar** [comment]
can I contribute? If yes, could you please tell me more about issue?

**Comment by saltcod** [comment]
@alaister wondering if we're loading a success message too optimistically here?
